docs/input.rst

```diff
@@ -203,7 +209,7 @@ The profiles given in the example are:
    \iota(\rho) &= 1 + 1.5 \rho^2.
    \end{aligned}
 
-If no profile inputs are given, it is assumed that they are :math:`p(\rho) = 0` and :math:`\iota(\rho) = 0`. 
+If no profile inputs are given, it is assumed that they are :math:`p(\rho) = 0` and :math:`\frac{2\pi}{\mu_0} I(\rho) = 0`. 
```

Comment:
defining I would be good to do here

Reply:
I think it's defined above on line 196. Here are the [docs built from this pr](https://desc-docs--251.org.readthedocs.build/en/251/input.html)